api/src/services/user.service.ts

```diff
@@ -193,7 +193,7 @@ export class UserService {
         },
       });
 
-      if (juris.allowSingleUseCodeLogin) {
+      if (juris?.allowSingleUseCodeLogin) {
```

Comment:
I feel like there shouldn't be a case where juris is undefined. What was the case that made this  change necessary?